pkg/boot/initrd_test.go

```diff
@@ -107,3 +108,64 @@ func TestCatInitrds(t *testing.T) {
 		}
 	}
 }
+
+func TestCreateInitrd(t *testing.T) {
+	for _, tt := range []struct {
```

Comment:
consider adding a filed called "name", to briefly describe current test case

then in test body, run it like,

t.Run(tt.name, func(t *testing.T){
      // copy your existing logic here.
})  

This help identify each individual test case in the log

Reply:
done!